kolibri/plugins/management/assets/src/vue/user-page/user-edit-modal.vue

```diff
@@ -20,20 +20,60 @@
         <div class="user-field">
           <label for="user-role"><span class="visuallyhidden">User Role</span></label>
           <select v-model="role_new" id="user-role">
-            <option value="learner" selected> Learner </option>
-            <option value="admin"> Admin </option>
+            <option :selected="role_new == admin ? true : false" v-if="role_new" value="learner"> Learner </option>
+            <option :selected="role_new == admin ? true : false" value="admin"> Admin </option>
           </select>
         </div>
 
+        <div class="advanced-options" v-if="!pw_reset && !usr_delete">
+          <button @click="pw_reset=!pw_reset"> Reset Password </button>
+          <button @click="usr_delete=!usr_delete"> Delete User</button>
```

Comment:
these should probably be be methods, rather than inline modifications to the data